torch/distributed/distributed_c10d.py

```diff
@@ -451,7 +466,8 @@ def default_pg(self, value):
     @property
     def pg_map(self) -> Dict[ProcessGroup, Tuple[str, Optional[Store]]]:
         """
-        Cached process groups
+        Process groups Cache.
```

Comment:
Not really a cache, but a mapping from ProcessGroup to backend name and store

Reply:
Hello @H-Huang thank you so much for the review and comments! 
For this particular change, I just tried to re-write the original doc string (the error was for using the word "Cached" so I re-write to overcome it). I will change it according to your suggestions.